src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -34,35 +34,68 @@ let nextId = 0;
 export class SkyCheckboxComponent implements ControlValueAccessor, OnInit {
 
   /**
-   * Hidden label for screen readers.
+   * Defines an ARIA label for the checkbox to support
+   * [accessibility](https://developer.blackbaud.com/skyux/components/checkbox#accessibility)
```

Comment:
This link works for now, but it points to the current page. I don't see an accessibility section in these updated docs, so this will be a busted link when we publish these changes. I think we should add the Accessibility section to the Design tab for now and just kill the link until we update `docs-tools` to enable linking between tabs.

Reply:
Added an Accessibility section to the Design tab.